### ISSUE TITLE
Hard to get back sound in JQBX after switching to Spotify, on Android

### ISSUE THREAD

**Issue by mikebeaton** [JasonZigelbaum/jqbx-issues#204]
TL/DR: If I switch from JQBX App to Spotify App and back, sound disappears on the JQBX App even though it still appears to be playing; Sync Sound does not help; I have to fully log out of Spotify and do some more steps in JQBX App (see below) to get sound back.

System: Samsung Galaxy S10 Lite, Samsung One UI 2.1, Android 10. JQBX 47.0.

Actions:
 - I log in to JQBX app, it works fine
 - I go into Spotify app and listen to something
 - I go back to JQBX app, I am still logged in and tracks appear to play, but there is no sound; Sync Sound does not bring sound back
 - I log out of JQBX app and log in again, still no sound; Sync Sound still does not help

Workaround:
 - Log out of Spotify app (fully log out, not just close)
 - Log out of JQBX app
 - Fully close JQBX app (this step seems to be required)
 - Re-open and log back in to JQBX app - at this point, I either have sound immediately, or else Sync Sound now works and brings the sound back immediately (I think both of these have happened)

The full workaround (all of which seems to be required) is rather non-obvious, and one or two steps too many to be able to work out at all easily!

I hope that's detailed enough and of some use - thanks for a great app and idea!

**Comment by mikebeaton** [comment]
Apologies, this appears to be the same issue as #85.
